charts/newrelic-infrastructure/tests/unprivileged_override_host_root_test.yaml

```diff
@@ -0,0 +1,33 @@
+suite: Test NRIA_OVERRIDE_HOST_ROOT
+templates:
+  - templates/kubelet/daemonset.yaml
+  - templates/kubelet/scraper-configmap.yaml
+  - templates/kubelet/agent-configmap.yaml
+  - templates/kubelet/integrations-configmap.yaml
+  - templates/agent-configmap.yaml
+  - templates/secret.yaml
+tests:
+  - it: NRIA_OVERRIDE_HOST_ROOT is not present in privileged mode
+    set:
+      licenseKey: test
+      cluster: test
+      privileged: true
+    asserts:
+      - notContains:
+          path: spec.template.spec.containers[1].env
+          content:
+            name: "NRIA_OVERRIDE_HOST_ROOT"
+            value: ""
+        template: templates/kubelet/daemonset.yaml
+  - it: NRIA_OVERRIDE_HOST_ROOT is present in unprivileged mode
+    set:
+      licenseKey: test
+      cluster: test
+      privileged: false
+    asserts:
+      - contains:
+          path: spec.template.spec.containers[1].env
+          content:
+            name: "NRIA_OVERRIDE_HOST_ROOT"
+            value: ""
+        template: templates/kubelet/daemonset.yaml
```

Comment:
Could you test `notContains` when privileged is set to false?

Reply:
... That was what this test was supposed to test. Now I wonder how it's passing.

Reply:
WDYM exactly? The env var _should_ be there (set to `""`) when `privileged: false`.